rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_motor_model.h

```diff
@@ -30,26 +30,30 @@
 #include <gazebo/common/Plugin.hh>
 #include <gazebo/gazebo.hh>
 #include <gazebo/physics/physics.hh>
-#include <mav_msgs/Actuators.h>
-#include <mav_msgs/default_topics.h>
-#include <ros/callback_queue.h>
-#include <ros/ros.h>
-#include <rotors_comm/WindSpeed.h>
-#include <std_msgs/Float32.h>
+
+#include <mav_msgs/default_topics.h>  // This comes from the mav_comm repo
 
 #include "rotors_gazebo_plugins/common.h"
 #include "rotors_gazebo_plugins/motor_model.hpp"
 
+#include "Float32.pb.h"
+#include "CommandMotorSpeed.pb.h"
+#include "WindSpeed.pb.h"
+
 namespace turning_direction {
 const static int CCW = 1;
 const static int CW = -1;
-}
+} // namespace turning_direction
 
 namespace gazebo {
+
 // Default values
 static const std::string kDefaultCommandSubTopic = "gazebo/command/motor_speed";
 static const std::string kDefaultWindSpeedSubTopic = "gazebo/wind_speed";
```

Comment:
`mav_comm` stuff should still work (I download the repo on the PX4 build also). I have now used constants from `mav_comm`.